README.md

```diff
@@ -63,12 +63,12 @@ dependencies[] = restful
  }
 ```
 
-####restful\_custom/plugins/restful/myplugin.inc
+####restful\_custom/plugins/restful/thenameoftheplugin.inc
```

Comment:
I was thinking along the same lines, however articles.inc is a little bit confusing.  It doe not emphasize that it has to be the same as the name of the plugin.   

Maybe changing it to articles.inc and having 'The name of your plugin must match your file' comment I added is good enough. 

If you are not happy with 'thenameoftheplugin' you could change it to your liking, like 'plugin_name'